packages/storage/src/storageBrowser/types.ts

```diff
@@ -90,3 +98,14 @@ export interface LocationCredentialsStore {
 	 */
 	destroy(): void;
 }
+
+/**
+ * Common interface for handlers to configure StorageBrowser behaviors
+ *
+ * @internal
+ */
+export interface StorageBrowserConfigAdapter {
```

Comment:
Since the adapter is not responsible for all `StorageBrowser` configuration properties prefer `StorageBrowserAuthConfigAdapter`:

```suggestion
export interface StorageBrowserAuthConfigAdapter {
```

But tbh not sure if "StorageBrowser" namespaces should live outside of the _ui-react-storage_ package. We should consider either finding a happy medium for the name that provides some context in to its purpose, or moving the adapter code in to the _ui-react-storage_ package

Reply:
@calebpollman  Are you OK we only export the managed auth adapter implementation and its input, output interfaces for now? The actual `StorageBrowserAuthConfigAdapter` interface to be shared across the Auth UCs still lives in UI package?

Reply:
Think that makes sense